cocotb_build_libs.py

```diff
@@ -19,6 +19,56 @@
 cocotb_share_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "cocotb", "share"))
 
 
+def create_rc_file(name, filename, libraries):
+    archtitecture = "amd64" if sys.maxsize > 2**32 else "x86"
+    dependencies = []
+
+    for lib in libraries:
+        dependencies.append(
+            '        "<dependency>"\n'
+            '            "<dependentAssembly>"\n'
+            '                "<assemblyIdentity name=""%s"" version=""1.0.0.0"" type=""win32"" processorArchitecture=""%s"" />"\n'
+            '            "</dependentAssembly>"\n'
+            '        "</dependency>"\n' % (lib, archtitecture))
+
+    rc_body = (
+        '#include <WinUser.h>\n'
+        '\n'
+        'LANGUAGE 0x00, 0x00\n'
+        '\n'
+        'ISOLATIONAWARE_MANIFEST_RESOURCE_ID RT_MANIFEST\n'
+        'BEGIN\n'
+        '    "<?xml version=""1.0"" encoding=""UTF-8"" standalone=""yes""?>"\n'
+        '    "<assembly xmlns=""urn:schemas-microsoft-com:asm.v1"" manifestVersion=""1.0"">"\n'
+        '        "<assemblyIdentity name=""%s"" version=""1.0.0.0"" type=""win32"" processorArchitecture=""%s"" />"\n'
+        '        "<file name=""%s"" />"\n'
+        '%s'
+        '    "</assembly>"\n'
+        'END\n' % (name, archtitecture, filename, "".join(dependencies)))
```

Comment:
Likely more readable with textwrap.dedent.
```suggestion
    rc_body = textwrap.dedent('''
        #include <WinUser.h>

        LANGUAGE 0x00, 0x00

        ISOLATIONAWARE_MANIFEST_RESOURCE_ID RT_MANIFEST
        BEGIN
            "<?xml version=""1.0"" encoding=""UTF-8"" standalone=""yes""?>"
            "<assembly xmlns=""urn:schemas-microsoft-com:asm.v1"" manifestVersion=""1.0"">"
                "<assemblyIdentity name=""%s"" version=""1.0.0.0"" type=""win32"" processorArchitecture=""%s"" />"
                "<file name=""%s"" />"
        %s
            "</assembly>"
        END''') % (name, archtitecture, filename, "".join(dependencies))
```

Reply:
I have changed it to textwrap.dedent(...)